src/grelu/interpret/motifs.py

```diff
@@ -21,60 +22,62 @@ def motifs_to_strings(
     Extracts a matching DNA sequence from a motif
```

Comment:
bit more docu: "Extracts a matching DNA sequence from a motif. Either the best match sequence or a sample from the PWM, depending on `sample`."

Reply:
done